pykeybasebot/kvstore_client.py

```diff
@@ -60,40 +61,39 @@ def __init__(self, bot):
             raise disambiguate_error(e)
 
     async def get(
-        self, team: str, namespace: str, entry_key: str
+        self, namespace: str, entry_key: str, team: Union[str, None] = None
     ) -> keybase1.KVGetResult:
         await self.bot.ensure_initialized()
-        res = await self.execute(
-            {
-                "method": "get",
-                "params": {
-                    "options": {
-                        "team": team,
-                        "namespace": namespace,
-                        "entryKey": entry_key,
-                    }
-                },
-            }
-        )
+        args: Dict[str, Any] = {
+            "method": "get",
+            "params": {"options": {"namespace": namespace, "entryKey": entry_key}},
+        }
+        if team:
+            args["params"]["options"]["team"] = team
+        res = await self.execute(args)
         return keybase1.KVGetResult.from_dict(res)
 
-    async def list_namespaces(self, team: str) -> keybase1.KVListNamespaceResult:
+    async def list_namespaces(
+        self, team: Union[str, None] = None
+    ) -> keybase1.KVListNamespaceResult:
         await self.bot.ensure_initialized()
-        res = await self.execute(
-            {"method": "list", "params": {"options": {"team": team}}}
-        )
+        args: Dict[str, Any] = {"method": "list", "params": {"options": {"team": team}}}
```

Comment:
```suggestion
        args: Dict[str, Any] = {"method": "list", "params": {"options": {}}}
```
no?

Reply:
this made me realize that the kb chat API deals with null values fine